uPortal-webapp/src/main/resources/layout/theme/respondr/respondr.xsl

```diff
@@ -652,7 +652,11 @@
             -->
             <chunk-point/> <!-- Performance Optimization, see ChunkPointPlaceholderEventSource -->
         </head>
-        <body class="up dashboard portal fl-theme-mist">
+        <body>
+          <xsl:attribute name="class">
+            <xsl:text>up dashboard portal fl-theme-mist</xsl:text>
+            <xsl:if test="$PORTAL_VIEW='focused'"> focused</xsl:if>
+          </xsl:attribute>
```

Comment:
I like the solution, generally speaking.  Should the '` focused`' characters also be wrapped in a `<xsl:text>`?  What's the purpose of that?

Reply:
[respondr.xsl#L156](https://github.com/Jasig/uPortal//blob/HEAD/uPortal-webapp/src/main/resources/layout/theme/respondr/respondr.xsl#L156) doesn't have have `<xsl:text>`. The reason for using `<xsl:text>` is to have the classes there regardless of the mode ... hopefully that answers the question